.travis.yml

```diff
@@ -54,11 +51,10 @@ jobs:
     - language: node_js
       node_js: 8.9
       cache:
-        yarn: true
       before_install:
         - cd superset/assets
       install:
-        - yarn install --frozen-lockfile
+        - npm install
```

Comment:
I think the proper command would be `npm ci` (see https://docs.npmjs.com/cli/ci.html) which is the equivalent of yarn's `frozen-lockfile`)